src/PowerShellEditorServices.VSCode/Cmdlets/VSCodeHtmlContentViewCommands.cs

```diff
@@ -53,25 +54,27 @@ protected override void ProcessRecord()
                 }
                 else
                 {
-                    throw new Exception("unable to fetch psEditor value");
+                    WriteError(
```

Comment:
Currently this is a non-terminating error.

Would it be better as a terminating error, since not being able to find the PSEditor object we won't be able to proceed meaningfully?

Terminating here wouldn't kill a script, just a pipeline, so it would operate with the desired effect I think.

```csharp
ThrowTerminatingError(
    new ErrorRecord(
        new ItemNotFoundException("Cannot find the '$psEditor' variable"),
        "PSEditorNotFound",
        ErrorCategory.ObjectNotFound,
        targetObject: null));
return;
```

Reply:
I think that makes sense. Updated!

Reply:
It doesn't kill the script, but in general I prefer non-terminating because that lets the user easily handle errors with the `-ErrorAction` common parameter.  With a terminating error, you *have* to `try`/`catch`, so I try to leave that for rare catastrophic errors.

I don't feel particularly strongly about not using terminating in this particular case though, just adding some more information.